pytorch_toolkit/formula_recognition/README.md

```diff
@@ -0,0 +1,214 @@
+- `device` - device for training, used in PyTorch .to() method. Possible options: 'cuda', 'cpu'. `cpu` is used by default.
+#### Training-specific parameters
+In addition to common parameters you can specify the following arguments:
+- `batch_size` - batch size used for training
+- `learning_rate` - learining rate
+- `log_path` - path to store training logs
+- `optimizer` - Adam or SGD
+- `save_dir` - dir to save checkpoints
+- `train_paths` - list of paths from where to get training data (if more than one path is specified, datasets are concatenated). If one wants to concatenate more than one instance of the desirable dataset, this dataset should be specified several times.
+- `val_path` - path to the validation data
+- `train_transforms_list` - similar to `val_transforms_list`
+- `epochs` - number of epochs to train
+
+One can use some pretrained models. Right now two models are available:
+* medium model:
+    * [checkpoint link](https://download.01.org/opencv/openvino_training_extensions/models/formula_recognition/medium_photograped_0185.pth)
+    * digits, letters, some greek letters, fractions, trigonometric operations are supported; for more details, please, look at [corresponding vocab file](vocabs/vocab_medium.json).
+    * to use this model, just set the correct value to the `model_path` field in the bcorresponding config file:
+    ```
+    model_path: <path to the model>
+    ```
+The model can be used for recognizing both rendered and scanned formulas (e.g. from a scanner or from a phone camera)
+
+* handwritten polynomials model:
+    * [checkpoint](https://download.01.org/opencv/openvino_training_extensions/models/formula_recognition/polynomials_handwritten_0166.pth)
+    * digits, letters, upper indices are supported
+    * to use this model, please, change model path in the corresponding config file:
+    ```
+    model_path: <path to the model>
+    ```
+The model can be used for recognizing handwritten polynomial equations.
+All the above models can be used for aftertuning or as ready for inference models. To provide maximum quality at recognizing formulas, it is highly recommended to preprocess image - simply binarize it:
+```
+val_transform_list:
+    - name: TransformBin
+      threshold: 100
+```
+You can find other prepocessing at [this file](im2latex/data/utils.py).
+Sample images in the [data](../../data) section of this repo are already preprocessed, you can look at the examples.
+
+
+#### Evaluation-specific parameters
+- `split_file` - name of the file with labels (note: physical file name should end with `_filter.lst`). Default is `validate`
+- `target_metric` - target value of the metric. Used in tests. For test to pass, result value should be greater or equal than `target_metric`
```

Comment:
not sure that it is needed in tests, it is needed for tests only.

Reply:
Removed from config file